_vector-search/ai-search/building-agentic-search-flows.md

```diff
@@ -25,27 +28,99 @@ To configure new agents, be sure to first provision appropriate models. For work
-    ![Agent searching]({{site.url}}{{site.baseurl}}/images/dashboards-flow-framework/agent-searching.png)
-1. Under **Generated query**, view the query domain-specific language (DSL) that the agent generated and ran against your cluster.
-    ![Agent query]({{site.url}}{{site.baseurl}}/images/dashboards-flow-framework/agentic-search-agent-query.png)
-1. Under **Search results**, select **Raw response**, the formatted table of **Hits**, or **Aggregations** to view the results.
-    ![Search results]({{site.url}}{{site.baseurl}}/images/dashboards-flow-framework/agentic-search-results.png)
+2. Provide a unique workflow **Name** and an optional **Description**, as shown in the following image. Then select **Create** to create your workflow. You are automatically directed to the workflow editor, where you can begin configuring the agent.
+   ![Quick configure modal]({{site.url}}{{site.baseurl}}/images/dashboards-flow-framework/agentic-search-quick-configure-modal.png)
+3. Under **Configure agent**, select **Create new agent**. Enter a unique **Name**, optionally provide a **Description**, and under **Model**, select the deployed **OpenAI GPT-5** model. Finally, select **Create agent** at the bottom.
+   ![Agentic search configuration]({{site.url}}{{site.baseurl}}/images/dashboards-flow-framework/agentic-search-example-configuration.png)
+4. Under **Agentic search**, select the index you'd like to search against. The agent will already be selected.
+5. Under **Query**, enter a natural language query about your data. Select **Search** to run an agentic search.
+   ![Agentic search]({{site.url}}{{site.baseurl}}/images/dashboards-flow-framework/agentic-search-example-search.png)
+6. View the agent's generated query, the search hits, and agent summary.
+   ![Agentic search query]({{site.url}}{{site.baseurl}}/images/dashboards-flow-framework/agentic-search-example-query.png)
+
+   ![Agentic search hits]({{site.url}}{{site.baseurl}}/images/dashboards-flow-framework/agentic-search-example-hits.png)
+
+   ![Agentic search summary]({{site.url}}{{site.baseurl}}/images/dashboards-flow-framework/agentic-search-example-summary.png)
+
+## Next steps
+
+- Try out Agentic Search right now on the [ML Playground](https://ml.playground.opensearch.org/app/opensearch-flow#/workflows/WAmxgJoBWVNV3bhKKnGx?configureAgent=false)
+- Check out the Agentic Search [documentation]({{site.url}}{{site.baseurl}}/vector-search/ai-search/agentic-search/) and [blog](https://opensearch.org/blog/introducing-agentic-search-in-opensearch-transforming-data-interaction-through-natural-language/)
+- Check out the OpenSearch agents [documentation]({{site.url}}{{site.baseurl}}/ml-commons-plugin/agents-tools/agents/index/)
+- Join the discussion on the [OpenSearch forum](https://forum.opensearch.org/)
```

Comment:
forum link https://forum.opensearch.org/t/use-cases-and-general-feedback-for-agentic-search/27488